enos/modules/docker_postgres/main.tf

```diff
@@ -63,6 +63,8 @@ resource "docker_container" "postgres" {
     "POSTGRES_USER=${var.user}",
     "POSTGRES_PASSWORD=${var.password}",
   ]
+  cpu_set = "1-2"
+  memory  = 2000
```

Comment:
During testing in CI, observed one instance where the worker test flaked due to our favorite `unexpected rsv bits` error (the test did pass on the retry). We previously addressed this issue by having the tests run on RUNNER_LARGE.

I haven't been able to reproduce the flake yet (re-running it ~10 times). This change here is one idea to try to limit the resources used by other containers.